Add CNF benefits as part of the WG scope (#9)
cncf/cnf-wg: charter.md

## In Scope
- Definition of Cloud Native Network Function (CNF)

>   - Provide a list of benefits of migrating to/implementing a Cloud-Native architecture

Reviewer: maybe change the scope of this to say list the cloud native benefits a best practice provides? I think for big picture items we could reference a lot of existing docs that OpenInfra, Linux Foundation and the big public cloud players.

Author: You mean like this?
```suggestion
  - Provide a list the cloud native benefits a best practice provides
```

Reviewer: I dig it.

Reviewer 2: I agree. If you change that, We can merge that
